basic_bootloader/src/bootloader/mod.rs

```diff
@@ -476,4 +487,77 @@ impl<S: EthereumLikeTypes> BasicBootloader<S> {
             }
         }
     }
+
+    #[cfg(feature = "interop")]
```

Comment:
Maybe we should just check that interop roots list is empty if interop is disabled?